app/routes/docs.producers._index.mdx

```diff
@@ -60,10 +66,18 @@ also happy to work with the mission teams to help construct your alerts.
       from gcn_kafka import Producer
       # Connect as a producer.
       # Warning: don't share the client secret with others.
-      producer = Producer(client_id='fill me in', client_secret='fill me in')
+      producer = Producer(client_id='fill me in',
+                          client_secret='fill me in')
+
       # any topic starting with 'mission.'
       topic = 'gcn.notices.mission.example'
-      data = b'...'  # any bytes
+
+      # JSON data converted to byte string format
+      data = str({
+         '$schema':'your/schema',
+         'key':'value'
+         ...
+      }).encode()  # any bytes
```

Comment:
No, that's not how you serialize a data structure to JSON in Python. Use [`json.dumps`](https://docs.python.org/3/library/json.html#json.dumps).

I don't think that you need to explicitly encode it to bytes. If memory servers, `Producer.produce` happily accepts either strings or bytes. (Please test that.)

Reply:
We will update to use json.dumps

We tested producer.produce with a string and it threw errors when not a byte string

@dakota002 please check this is true